Tests/GRPCTests/GRPCReflectionServiceTests/ReflectionServiceUnitTests.swift

```diff
@@ -117,134 +117,199 @@ final class ReflectionServiceUnitTests: GRPCTestCase {
   func testNameOfFileContainingSymbolEnum() throws {
     let protos = makeProtosWithDependencies()
     let registry = try ReflectionServiceData(fileDescriptors: protos)
-    let fileName = registry.nameOfFileContainingSymbol(named: "packagebar2.enumType2")
-    XCTAssertEqual(fileName, "bar2.proto")
+    let nameOfFileContainingSymbolResult = registry.nameOfFileContainingSymbol(
+      named: "packagebar2.enumType2"
+    )
+    switch nameOfFileContainingSymbolResult {
```

Comment:
No need to change these now, but it would've been much easier to just do:

```swift
XCTAssertEqual(try nameOfFileContainingSymbolResult.get(), "bar2.proto")
```